apps/dashboard/src/components/ipfs-upload/dropzone.tsx

```diff
@@ -429,11 +429,11 @@ const FileUpload: React.FC<FileUploadProps> = ({ files, updateFiles }) => {
             >
               <Progress
                 colorScheme="green"
-                value={progress.progress ? progressPercent : undefined}
+                // value={progress.progress ? progressPercent : undefined}
                 size={{ base: "xs", md: "lg" }}
                 w="100%"
                 borderRadius="full"
-                isIndeterminate={progress.progress === progress.total}
+                // isIndeterminate={progress.progress === progress.total}
```

Comment:
just set this `true` while the upload mutation is pending for now (which will at least show the bar moving left and right while upload is happening)